### DISCUSSION TITLE
火狐浏览器onClick跳转到第三方页面不生效，直接刷新了当前页面

### DISCUSSION THREAD

**Discussion by Payzlee001** [umijs/umi#11858] [Q&A]
umi版本号：4.0.88
react：18.0.33
使用npm create umi 创建的项目，直接项目启动示例代码的Github跳转就无效；
示例代码：
import { Link, Outlet } from 'umi';
import styles from './index.less';

export default function Layout() {
  return (
    <div className={styles.navs}>
      <ul>
        <li>
          <Link to="/">Home</Link>
        </li>
        <li>
          <Link to="/docs">Docs</Link>
        </li>
        <li>
          <a href="https://github.com/umijs/umi">Github</a>
        </li>
      </ul>
      <Outlet />
    </div>
  );
}

尝试过改为onClick （windo.location.href 、window.open等）都不行，但onClick事件是触发了的，打印有数据。

**Comment by Payzlee001** [comment]
其他浏览器都可以

**Comment by fz6m** [comment]
同 #10691 ，我也不知道为什么，先用 chrome 开发吧，构建完是没问题的，如有兴趣可以帮助探索原因。

**Comment by Payzlee001** [comment]
该问题已经解决了，在package.json文件中修改启动命令 "dev": "cross-env HMR=none npm run start:dev",主要是cross-env HMR=none这个加上就解决了
